qemu/tests/watchdog.py

```diff
@@ -283,16 +283,18 @@ def hotplug_unplug_watchdog_device():
     # main procedure
     test_type = params.get("test_type")
     error.context("'%s' test starting ... " % test_type, logging.info)
-    error.context("Boot VM with WDT(Device:'%s', Action:'%s'),and try to login"
-                  % (watchdog_device_type, watchdog_action), logging.info)
-    params["start_vm"] = "yes"
-    env_process.preprocess_vm(test, params, env, params.get("main_vm"))
-    vm = env.get_vm(params["main_vm"])
-    session = vm.wait_for_login(timeout=timeout)
-
-    if params.get("setup_runlevel") == "yes":
-        error.context("Setup the runlevel for guest", logging.info)
-        utils_test.qemu.setup_runlevel(params, session)
+
```

Comment:
I think drop line#4 and append below lines in line#18 of watchdog.cfg can fix bug you meet.

Thanks,
Xu